### PR TITLE
Update InputEmulation.py

### PR DESCRIPTION
**Pull request by YamazakyLau**
Release the mouse button before 'raise IOError', When the system is busy.

### CODE REVIEW

File: lackey/InputEmulation.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def moveSpeed(self, location, seconds=0.3):
         original_location = mouse.get_position()
         mouse.move(location.x, location.y, duration=seconds)
         if mouse.get_position() == original_location and original_location != location.getTuple():
+            self._lock.release()
```
Comment (glitchassassin) [inline lackey/InputEmulation.py:69]: This can be simplified by moving it above the if statement and eliminating line 74.

**Review by glitchassassin — CHANGES_REQUESTED** [review]
I have just one minor change, thanks for your contribution!

File: lackey/InputEmulation.py (hunk 1 above)
Comment (FranciscoACLima) [reply lackey/InputEmulation.py:69]: Hi.
If line 74 "self._lock.release()" is removed, the lackey stops working.
The solution to insert self._lock.release() before the IOError raise was great.
Thank you again!

Comment (FranciscoACLima) [reply lackey/InputEmulation.py:69]: Hi.
I'm sorry, but
I hadn't tried moving line 69 out of the if.
I tested it again. Deleting line 74 worked.

**Comment by glitchassassin** [comment]
Resolved with #170.

**Pull request closed without merging**